lib/lob/client.rb

```diff
@@ -4,6 +4,7 @@
 require "lob/resources/group"
 require "lob/resources/intl_verifications"
 require "lob/resources/letter"
+require "lob/resources/member"
```

Comment:
Just a thought - would it be more clear if we named these endpoints `groups_members`?